diagnostic.md

```diff
@@ -32,23 +35,28 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
-
+$("li.listerItem").html is a getter
+$("li.listerItem").html("Listed item") is a setter
+$("#loginform").text() is a getter
+$("button").class("#redbutton") is a setter (I think, maybe I did this wrong?)
```

Comment:
In order for this to be a setter, you would want to use something like `addClass()`